app/src/main/java/com/serwylo/lexica/game/Game.java

```diff
@@ -44,17 +44,31 @@
 
 public class Game implements Synchronizer.Counter {
 
+	public static final String SCORE_TYPE = "scoreType";
+	public static final String SCORE_WORDS = "W";
+	public static final String SCORE_LETTERS = "L";
+
 	private static final String TAG = "Game";
 	private int timeRemaining;
 	private int maxTime;
 
 	private int maxTimeRemaining;
 
 	private Board board;
+	private int score;
+	private String scoreType;
 
 	public enum GameStatus { GAME_STARTING, GAME_RUNNING, GAME_PAUSED, GAME_FINISHED }
 
-	public static final int WORD_POINTS[] = {
+	//TODO: i18n
```

Comment:
It isn't important with regards to this PR, however I am working on i18n in my personal `languages` branch which will be ready to merge soon. Already have support for Spanish, German, Italian, Dutch, and Catalonian. I look forward to getting some time to polish and merge that in the future.

Reply:
Yeah, here's where this starts to get tricky. I wanted to get these basic changes out of the way before committing my own i18n branch in my fork. I also have improved and changed the Trie representation and board generation code to be able to handle whatever dictionary you can throw at it, along with generating appropriate letter points based on frequencies. Unfortunately, I started on it before realizing you had also made a branch for that work. Our work differs quite a bit, so rather than a pull request, I'll probably just point you to my branch and let you judge for yourself what the final code ends up like. Definitely wanted to get that code cleanly separated from these other changes first, though. Again, not germane to this request, but I consider this LETTER_POINTS[] quite temporary.

Reply:
Oh, that is interesting. Indeed the only reason I introduced the `StringTrie` was because the previous version was so hard-coded towards an English alphabet. Required ripping the entire guts out of the game and replacing it. 

I wish I had of merged my language stuff earlier. The only thing holding it back was the generation of letter probabilities, but even then I have a genetic-algorithm which can run and generate semi-reasonable letter distributions for arbitrary dictionaries also.

I'll make sure to check out what you've done, and I apologies for not making my progress more clear on the issue tracker, perhaps resulting in your doing a lot work for it not to be merged.

Reply:
For your reference, the commit for my i18n work is here: https://github.com/JulianPaoloThiry/lexica/commit/0893949068ebfed0710e08e38bfe7f3404566c90.
Feel free to use or not use all or some of it as you see fit.